app/src/main/java/com/conupods/OutdoorMaps/MapInitializer.java

```diff
@@ -0,0 +1,66 @@
+package com.conupods.OutdoorMaps;
+
+import android.graphics.Color;
+import android.view.KeyEvent;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.conupods.R;
+
+import static android.view.inputmethod.EditorInfo.IME_ACTION_DONE;
+import static android.view.inputmethod.EditorInfo.IME_ACTION_SEARCH;
+
+public class MapInitializer {
+
+    CameraController mCameraController;
+    private static final String TAG = "MapInitializer";
+
+    public MapInitializer(CameraController CameraController) {
+        mCameraController = CameraController;
+    }
+
+    public void initializeToggleButtons(Button sgwButton, Button loyButton) {
+        // The two campus swap buttons
```

Comment:
sometimes theres a space here and sometimes there isn't. We'll need to add a convention for this